setup.cfg

```diff
@@ -1,3 +1,66 @@
+[metadata]
+name = PyVISA
+author = Torsten Bronger, Gregor Thalhammer
+author_email = bronger@physik.rwth-aachen.de
+maintainer = Matthieu C. Dartiailh
+maintainer_email = m.dartiailh@gmail.com
+license = MIT License
+description = Python VISA bindings for GPIB, RS232, TCPIP and USB instruments
+keywords =
+    VISA
+    GPIB
+    USB
+    serial
+    RS232
+    measurement
+    acquisition
+url = https://github.com/pyvisa/pyvisa
+long_description = file: README.rst, AUTHORS, CHANGES
+long_description_content_type = text/x-rst
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX :: Linux
+    Operating System :: MacOS :: MacOS X
+    Programming Language :: Python
+    Topic :: Scientific/Engineering :: Interface Engine/Protocol Translator
+    Topic :: Software Development :: Libraries :: Python Modules
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+platforms = Linux,, Windows,, Mac
+
+[options]
+py_modules = visa
+packages =
+    pyvisa
+    pyvisa.ctwrapper
+    pyvisa.resources
+    pyvisa.thirdparty
+    pyvisa.testsuite
+    pyvisa.testsuite.fake-extensions
+    pyvisa.testsuite.fakelibs
+    pyvisa.testsuite.keysight_assisted_tests
+zip_safe = False
+install_requires =
+python_requires = >=3.6
+use_2to3 = False
+platforms = Linux; Windows; Mac
+test_suite = pyvisa.testsuite.testsuite
+setup_require = setuptools>=42; wheel; setuptools_scm[toml]>=3.4.3
+
+[options.entry_points]
+console_scripts =
+    pyvisa-shell=pyvisa.cmd_line_tools:visa_shell
+    pyvisa-info=pyvisa.cmd_line_tools:visa_info
+
+[options.package_data]
+* = *.dll
```

Comment:
Why is this line here we do not ship any pre-built dll ?

Reply:
I have only translated `setup.py` + some minor changes related to packaging. I also think that wheels should contain no dlls. But I estimated that the chances are higher to get this PR merged if it is not changing anything that might be controversal. So  I haven't removed the dlls. Especially since I cannot test it now since I'm on a linux machine.

Reply:
My bad I have some fake dll files to test reading the headers. So it should stay. I will merge soon.